metadata-ingestion/transformers.md

```diff
@@ -6,6 +6,12 @@ Oftentimes we want to modify metadata before it reaches the ingestion sink – f
 
 Moreover, a transformer allows one to have fine-grained control over the metadata that’s ingested without having to modify the ingestion framework's code yourself. Instead, you can write your own module that can take MCEs however you like. To configure the recipe, all that's needed is a module name as well as any arguments.
 
+## Note about transformers use
+
+The naming of the transformers is based on what they do to the data created by ingestion sources. It is not based on what finally happens in datahub. e.g. `simple_add_dataset_tags` adds a set of tags to the data created by ingestion sources. So if source did not add any tags you can use `simple_add_dataset_tags` to add any tags using this transformer. But this will overwrite any tags already present in the system. If you have done an ingestion once and then added some tags through the UI this will overwrite those tags. 
```

Comment:
Unfortunately, this is not true about all things today. 
e.g. Tags and Descriptions have EditableSchemaMetadata aspect that is written to from the UI, thus keep the "system ingestion" pathway separate from the UI pathway. 

Other things like Ownership don't have the same split, so they can get overwritten by ingestion pathway.

Reply:
This confusion can cause some nasty surprises